apps/studio/components/layouts/ProjectLayout/LayoutHeader/LocalVersionPopover.tsx

```diff
@@ -0,0 +1,157 @@
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="macos">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="windows">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    scoop upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="linux">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    brew upgrade supabase
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+                <TabsContent_Shadcn_ className="mt-2 text-xs" value="npm">
+                  <SimpleCodeBlock parentClassName="bg-selection rounded !px-2">
+                    npm update supabase --save-dev
+                  </SimpleCodeBlock>
+                </TabsContent_Shadcn_>
+              </Tabs_Shadcn_>
+            </div>
+          ) : null
+        ) : null}
+
+        <div className="flex flex-col gap-y-2 mt-3">
+          <p className="text-xs text-foreground-lighter">
+            All available image versions of Supabase Studio can be found on our{' '}
+            <InlineLink href="https://hub.docker.com/r/supabase/studio/tags">Docker Hub</InlineLink>
+            .
+          </p>
+        </div>
+
+        <div className="flex items-center gap-x-2 mt-3">
+          <Dialog>
+            <DialogTrigger asChild>
+              <Button type="default">Release schedule</Button>
+            </DialogTrigger>
+            <DialogContent>
+              <DialogHeader className="border-b">
+                <DialogTitle>Stable release schedule</DialogTitle>
+              </DialogHeader>
+              <DialogSection className="flex flex-col gap-y-2">
+                <p className="text-foreground-lighter text-xs font-mono uppercase">
+                  Approximate next release: {approximateNextRelease}
+                </p>
+                <p className="text-sm">
+                  Supabase CLI releases follows a 2 weeks cadence, which stable releases will be
+                  available for{' '}
+                  <InlineLink href="https://supabase.com/docs/guides/local-development/cli/getting-started?queryGroups=platform&platform=linux#updating-the-supabase-cli">
+                    update
+                  </InlineLink>{' '}
+                  via the CLI.
+                </p>
+                <Admonition
+                  type="default"
+                  title="Beta Releases"
+                  description="Beta releases are also available in between stable releases via the Beta version of the CLI if you might be waiting on a fix. If you'd like to try, we recommend doing so via npm:"
```

Comment:
Beta releases are also available between stable releases through the Beta version of the CLI, which can be helpful if you're waiting for a specific fix. 

?